### DISCUSSION TITLE
좋은 스터디 그룹에 대한 글 (스터디 그룹 패턴)

### DISCUSSION THREAD

**Discussion by fkdl0048** [ThinkAboutSoftware/AcademicConference#400] [General]
# 좋은 스터디 그룹에 대한 글 (스터디 그룹 패턴)

최근에 스터디에 대한 좋은 글을 공유 받아서 아카데미 컨퍼런스에도 공유합니다. 아마 이미 읽으신 분들이나 아시는 분들도 많을 것 같지만, 실패하는 스터디에서 공통적으로 나오는 문제점들이나 좋은 스터디의 특징들을 잘 알 수 있어서  좋은 글인 것 같습니다.

견습자 패턴과 같이 읽기 쉽게 구성되어 있으니 한번 읽어보고 아카데미 컨퍼런스에 대해서도 생각해보면 좋을 것 같습니다. 이미 잘 선행되고 있는 내용도 있고, 적용해보면 좋을 것 같은 내용도 있어서 같이 토론해보고 개선점을 찾아봐도 좋을 것 같습니다..!

- [스터디 그룹 패턴 번역 글](https://lifeiscool.tistory.com/86)

**Comment by jongfeel** [comment]
일단 저는 몰랐던 내용이었고 제가 지금 하고 있는 모임은 저의 실천 적인 경험을 바탕으로 만들어진 시스템이다 보니
올바른 방향으로 가는 데 사람들이 하는 생각은 비슷하구나를 다시 한번 느끼게 됐습니다.

읽어 봤는데 제가 academic conference에서 실천하고 있는 내용도 있네요.

4번의 지속적인 에너지는 제가 주도하고 있고 열정적이기도 해서 거의 같은 것 같고요.
5번 마음 맞는 사람들
6번 공동의 장소
이 항목들도 어느 정도 맞는 것 같습니다.
